scripts/mediaViews.php

```diff
@@ -11,19 +11,25 @@
 $db = Doctrine_Manager::getInstance()->getCurrentConnection();
 
 //prune logs
-$date = date('Y-m-d H:i:s', strtotime('1 month ago'));
-
-$sql = "DELETE FROM media_views WHERE datecreated < ?";
+$sql = "DELETE FROM media_views WHERE datecreated < (NOW() - INTERVAL 1 MONTH)";
```

Comment:
The `media_views` table just houses a record for each view to a piece of media for the last 30 days (date of view, IP address, etc). It serves as a way for us to calculate recent popularity and also prevent the spamming of views from the same IP address. 

Deleting all data over 30 days old is just meant to keep the table small.